modules/querier/worker/worker.go

```diff
@@ -242,6 +249,9 @@ func (w *querierWorker) resetConcurrency() {
 	if totalConcurrency > w.cfg.MaxConcurrentRequests {
 		level.Warn(w.log).Log("msg", "total worker concurrency is greater than promql max concurrency. Queries may be queued in the querier which reduces QOS")
 	}
+
+	// capture the current concurrency metric
+	metricConcurrency.Set(float64(totalConcurrency))
```

Comment:
why don't we just log this if we're concerned it's a different value then expected.

Reply:
updated.